### PR TITLE
Supporting iOS 7 #3

### PR DESCRIPTION
**Pull request by serejahh**

### CODE REVIEW

**Comment by maksym-letiushov** [comment]
@serejahh when I just switch deployment target to iOS 7.1 I am able to run the project, but as I see something is wrong with layout. Could you plz take a look http://monosnap.com/image/cLRgAj5q5HujorKVfHQF3J5P9GZTgr
I mean an extra black areas on the bottom and on the top of the screen.
Have you faced the same problem ?
Thank you for contribution

**Comment by serejahh** [comment]
@maksimusenko, yes, I've reproduced it. iOS 7 doesn't use LaunchScreen.xib. You have to use assets catalog or an image with postfix "-568h" to adapt an app for iPhone 5 with iOS 7. I've fixed it (a pull request will be later)

**Comment by maksym-letiushov** [comment]
@serejahh thanks
Make appropriate fixes and push them to this branch. Let me know when you are done.
Also please update tag in the FoldingTabBar.podspec to 1.0.1
Then I will submit the new version to the CocoaPods

**Comment by serejahh** [comment]
@maximletushov, it's done

File: Example/Example.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -563,8 +563,9 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				ASSETCATALOG_COMPILER_LAUNCHIMAGE_NAME = LaunchImage;
```
Comment (maksym-letiushov) [inline Example/Example.xcodeproj/project.pbxproj:566]: thanks

File: FoldingTabBar.podspec [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |spec|
   spec.name = "FoldingTabBar"
-  spec.version = "1.0.0"
+  spec.version = "1.0.1"
```
Comment (maksym-letiushov) [inline FoldingTabBar.podspec:3]: thanks

File: FoldingTabBar.podspec [hunk 3]
```diff
@@ -9,10 +9,10 @@ Pod::Spec.new do |spec|
   spec.license = { :type => "MIT", :file => "LICENSE" }
   spec.social_media_url = "https://twitter.com/yalantis"
 
-  spec.platform = :ios, '8.0'
-  spec.ios.deployment_target = '8.0'
+  spec.platform = :ios, '7.0'
+  spec.ios.deployment_target = '7.0'
 
-  spec.source = { :git => "https://github.com/Yalantis/FoldingTabBar.iOS.git", :tag => "1.0.0" }
+  spec.source = { :git => "https://github.com/Yalantis/FoldingTabBar.iOS.git", :tag => "1.0.1" }
```
Comment (maksym-letiushov) [inline FoldingTabBar.podspec:15]: thanks

File: README.md [hunk 4]
```diff
@@ -12,13 +12,13 @@ Also, read how it was done in our [blog](http://yalantis.com/blog/how_we_created
...
 ## Requirements
-iOS 8.0
+iOS 7.0
 
 ## Installation
 
 ####[CocoaPods](http://cocoapods.org)
 ```ruby
-pod 'FoldingTabBar', '~> 1.0.0'
+pod 'FoldingTabBar', '~> 1.0.1'
```
Comment (maksym-letiushov) [inline README.md:21]: thanks

**Pull request merged**